src/main/java/mate/academy/service/MovieServiceImpl.java

```
@@ -0,0 +1,22 @@
+package mate.academy.service;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService{
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id).orElseThrow();
```

let's actually throw an exception in your orElseThrow()